src/pages/Hosts/index.js

```diff
@@ -16,22 +16,21 @@ const PAGE_SIZE = 10;
 export default function Hosts() {
   const history = useHistory();
   const evernode = useEvernode();
+  const {nextPageToken, onChangeNextPageToken, pageQueue, onChangePageQueue} = useEvernode();
```

Comment:
Shall we rename `onChangeNextPageToken` to `updateNextPageToken` and `onChangePageQueue` to `updatePageToken`